code/game/atoms_movable.dm

```diff
@@ -833,11 +833,21 @@
 				break
 	. = dense_object_backup
 
-///called when a mob resists while inside a container that is itself inside something.
-/atom/movable/proc/relay_container_resist_act(mob/living/user, obj/O)
+/**
+  * Called when a mob resists while inside a container that is itself inside something.
+  *
+  * Arguments:
+  * * resistor - Mob calling the container_resist_act
+  * * container - containing object the mob is nested within
+  */
+/atom/movable/proc/relay_container_resist_act(mob/living/resistor, obj/container)
+	//Generic case for mob_holders
+	if(istype(container, /obj/item/clothing/head/mob_holder))
```

Comment:
Couldn't you just make this be

Reply:
This actually is in essence what that proc does currently....

However, container_resist_act doesn't proc recursively and the mob_holder is considered its own atom. So if the atom needs to make any checks for a container_resist_act on a mob, the holder has to relay it, which is what this is for.